pkg/c3/defs.h

```diff
@@ -21,34 +21,6 @@
 
   /** Random useful C macros.
   **/
-    /* Assert.  Good to capture.
-
-       TODO: determine which c3_assert calls can rather call c3_dessert, i.e. in
-       public releases, which calls to c3_assert should abort and which should
-       no-op? If the latter, is the assert useful inter-development to validate
-       conditions we might accidentally break or not useful at all?
-    */
-
-#     if defined(ASAN_ENABLED) && defined(__clang__)
-#       define c3_assert(x)                       \
-          do {                                    \
-            if (!(x)) {                           \
-              assert(x);                          \
-            }                                     \
-          } while(0)
-#     else
-#       define c3_assert(x)                       \
-          do {                                    \
-            if (!(x)) {                           \
-              fflush(stderr);                     \
-              fprintf(stderr, "\rAssertion '%s' " \
-                      "failed in %s:%d\r\n",      \
-                      #x, __FILE__, __LINE__);    \
```

Comment:
__LINE__ can technically overflow :^) I've seen it strigified for this reason. But personally don't care about this. Probably only relevant for generated code

Reply:
If we have over two billion lines (looks like `__LINE__` is `int` according to https://github.com/tloncorp/ylem/issues/822) in a single file, then we have bigger problems.

Reply:
lol agree this is absolutely pedantic